leasing/fixtures/batchrun_command.json

```diff
@@ -56,7 +56,47 @@
       "type": "django-manage",
       "name": "send_invoices_to_laske",
       "parameters": {},
-      "parameter_format_string": ""
+      "parameter_format_string": "1"
+    }
+  },
+  {
+    "model": "batchrun.command",
```

Comment:
I am wondering whether the command should be registered once, but multiple jobs created with different arguments. Here you can't seemingly name the different commands, they have the same name, it is going to be difficult to differentiate.
What is the difference between command and job, and can there be multiple commands with the same name, and if there can, how can a user differentiate them e.g. in django admin?

Reply:
I delved into the code more and I believe its possible to pass arguments to command so I changed it so that there is only 1 command that the jobs pass different arguments to